package.json

```diff
@@ -20,24 +26,37 @@
       "url": "https://github.com/cowboy/node-findup-sync/blob/master/LICENSE-MIT"
     }
   ],
-  "main": "lib/findup-sync",
-  "engines": {
-    "node": ">= 0.6.0"
-  },
+  "files": [
+    "index.js"
+  ],
+  "main": "index.js",
   "scripts": {
-    "test": "grunt nodeunit"
+    "test": "grunt && mocha -R spec"
+  },
+  "engines": {
+    "node": ">= 0.8.0"
   },
   "dependencies": {
-    "glob": "~4.3.0"
+    "is-glob": "^1.1.0",
+    "micromatch": "^1.2.2"
   },
   "devDependencies": {
-    "grunt": "~0.4.4",
-    "grunt-contrib-jshint": "~0.9.2",
-    "grunt-contrib-nodeunit": "~0.3.3"
+    "grunt": "^0.4.5",
+    "grunt-contrib-jshint": "^0.11.0",
+    "grunt-mocha": "^0.4.11",
```

Comment:
Good catch. I had grunt mocha hooked up and removed it last minute. 

Sent from my iPhone

> On Jan 31, 2015, at 3:39 PM, Shinnosuke Watanabe <notifications@github.com> wrote:
> 
> In package.json:
> 
> >    },
> >    "devDependencies": {
> > -    "grunt": "~0.4.4",
> > -    "grunt-contrib-jshint": "~0.9.2",
> > -    "grunt-contrib-nodeunit": "~0.3.3"
> > +    "grunt": "^0.4.5",
> > +    "grunt-contrib-jshint": "^0.11.0",
> > +    "grunt-mocha": "^0.4.11",
> grunt-mocha isn't used in Gruntfile.
> 
> —
> Reply to this email directly or view it on GitHub.
>